package.json

```diff
@@ -61,6 +61,7 @@
     "request": "~2.88.2",
     "semver": "~7.3.5",
     "simple-icons": "5.1.0",
+    "use-debounce": "^7.0.0",
```

Comment:
This should be moved to `devDependencies` since it's not needed in production on the server.

Reply:
This looks like it's still outstanding ^^